lib/bitcoin/cw_bitcoin.dart

```diff
@@ -522,7 +522,8 @@ class CWBitcoin extends Bitcoin {
   }
 
   @override
-  int getHeightByDate({required DateTime date}) => getBitcoinHeightByDate(date: date);
+  Future<int> getHeightByDate({required DateTime date}) async =>
+      await getBitcoinHeightByDateAPI(date: date);
```

Comment:
same,
this will be called if the user has the settings enabled, else we can keep using the old function